_config.yml

```diff
@@ -8,18 +8,12 @@ languages: ["fr", "en", "es", "es-ct"]
 base-url: ""
 
 # Social media
-twitter: # Card customization
```

Comment:
That is weird to have those lines as modification.
You might need to start this PR from the most recent version of master branch
because the modif here came from your last PR